Changed the color, code cleanup

ScratchnSniff0 opened the pull request:
Fixed the colors, remodeled some buttons, cleaned up some unnecessary code (various imports not used).  I need everybody's most recent work though to really remodel and unify the design/layout.

caydenreynolds commented on simulationfactory/src/util/Stylesheet.js:
```diff
@@ -1,55 +1,52 @@
+      padding: Theme.spacing(3),
     },
     defaultButton: {
       variant: "contained",
-      color: "primary",
       size: "medium"
     },
```
This class has no effect on the appearance of the buttons. We should probably remove it, and stop referencing it on all the buttons.
I'm like 90% sure I made this class, and forgot to delete it. That's my bad.

ScratchnSniff0 replied:
Good to go, I will take a look at it either later this evening or tomorrow morning!

caydenreynolds commented on simulationfactory/src/util/theme.html:
```diff
@@ -0,0 +1,193 @@
+<div class="mdc-components">
```
What is this file? Do we need it? How do we access it from the website?

ScratchnSniff0 replied:
This was one of my attempts at importing a theme that I forgot to remove.  We definitely don't need it. I'll figure out how to remove it later today, or early tomorrow.

ScratchnSniff0 commented:
I removed that code, did a little more remodeling, got some things to line up better. Added the about page mockup, wasn't sure if anybody DIDN'T want to be in it so I left everybodys names out until we can ask them. The github link is just a placeholder to ycps main page.
